vendor/github.com/olivere/elastic/client.go

```diff
@@ -381,7 +381,7 @@ func SetHealthcheckInterval(interval time.Duration) ClientOptionFunc {
 
 // SetMaxRetries sets the maximum number of retries before giving up when
 // performing a HTTP request to Elasticsearch.
-func SetMaxRetries(maxRetries int) func(*Client) error {
+func SetMaxRetries(maxRetries int) ClientOptionFunc {
```

Comment:
Create https://github.com/kubernetes/heapster/pull/1202 to take care this comment.